lib/livebook_web/live/settings_live.ex

```diff
@@ -119,6 +119,15 @@ defmodule LivebookWeb.SettingsLive do
                   name="editor_high_contrast"
                   label="Use high contrast theme"
                   checked={false} />
+                <div class="flex space-x-3 items-center justify-between">
+                  <span class="text-gray-700">Controls how lines should wrap</span>
+                  <div>
+                    <.select
+                    name="editor_word_wrap"
+                    options={[off: "off", on: "on", wordWrapColumn: "wordWrapColumn", bounded: "bounded"]}
```

Comment:
We should use a human readable descriptions of he other options, so the distinction is clear.

I agree with @josevalim that we should investigate configuring this temporarily editor-level is possible (through options menu).